spectral_test_files/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
```

Comment:
To make everything in this directory ignored, but not the directory itself. If I understand it correctly this is for temporary files produced by tests. For data needed by tests there is spectral/tests/data.

Reply:
I've gave it a second thought and change it to be ignored in global .gitignore. This makes it trivial to delete whole spectral_test_files directory when needed.